cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -263,15 +264,26 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
             ortResultInput = ortResultInput.replacePackageCurations(curations)
         }
 
+        val config = globalOptionsForSubcommands.config
+
+        val packageConfigurations =
+            packageConfigurationOption.createProvider().getPackageConfigurations().toMutableList()
         val repositoryPackageConfigurations = ortResultInput.repository.config.packageConfigurations
-        val optionPackageConfigurations = packageConfigurationOption.createProvider().getPackageConfigurations()
 
-        val packageConfigurationProvider = SimplePackageConfigurationProvider(
-            optionPackageConfigurations + repositoryPackageConfigurations
-        )
+        if (config.evaluator.applyRepositoryPackageConfigurations) {
+            packageConfigurations += repositoryPackageConfigurations
+        } else {
+            if (repositoryPackageConfigurations.isNotEmpty()) {
+                log.warn {
+                    "Package-configurations from '$ORT_REPO_CONFIG_FILENAME' not applied because the feature is not" +
```

Comment:
I believe that both assumptions about the filenames are wrong.
The `.ort.yml` could also be called `my-project.ort.yml`. 
IIRC also the ort config filename can be overridden.
How about dropping the filename?